livestream/models.py

```diff
@@ -16,4 +9,51 @@ class Archive(models.Model):
     def __str__(self):
         return self.archive_id
 
-# Create your models here.
+
+class Product(models.Model):
+    owner = models.OneToOneField(User,
+                                 on_delete=models.CASCADE,
+                                 related_name='product')
+
+    name = models.CharField(max_length=30)
+    description = models.CharField(max_length=500)
+    date_sold = models.DateTimeField()
+    price = models.PositiveIntegerField()
+    winning_bid = models.PositiveIntegerField()
+    minimum_price = models.PositiveIntegerField()
+
+
+class Bid(models.Model):
+    owner = models.OneToOneField(User,
+                                 on_delete=models.CASCADE)
+    credits = models.PositiveIntegerField()
+
+
+# Stores newly generated session id
+class Session(models.Model):
+    auctioneer = models.OneToOneField(User,
+                                      on_delete=models.CASCADE,
+                                      related_name='current_auctioneer')
+
+    # not sure
+    attendees = models.ForeignKey(User,
+                                  on_delete=models.CASCADE,
+                                  related_name='attendee')
+
+    products = models.ForeignKey(Product,
+                                 on_delete=models.CASCADE,
+                                 related_name='product')
+
+    bid = models.OneToOneField(Bid,
+                               on_delete=models.CASCADE,
+                               related_name='bid')
+
+    session_id = models.CharField(max_length=1000)
```

Comment:
This seems to be too much for a session id length. Find out the format first. Is it [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier)? If it is, what variant of UUID? These are standards and have fixed lengths.